KNeighbor-yigalron.py

```diff
@@ -0,0 +1,190 @@
+
```

Comment:
This file should be moved to `algorithms/scripts/KNeighbor-yigalron.py`.

I think you can do `git mv KNeighbor-yigalron.py algorithms/scripts/KNeighbor-yigalron.py`. Also you'll need to move `KNeighbor-yigalron.ipynb` to `algorithms/KNeighbor-yigalron.ipynb`.